README.md

```diff
@@ -174,7 +174,7 @@ CSS snippets are rather basic to implement. We fetch them from this repo, so you
 1. `componentDidMount` triggers `newRequest`, which triggers `loadAmount(30)`
 2. `loadAmount` calls `loadPage` in a loop until it has the requested amount of cards or runs out of results
 3. `loadPage` calls `getRepos(page)` to get the next page of extensions. It queries the GitHub API for any repos with the "spicetify-extensions" topic. We'll likely add our own tag in the future, like "spicetify-marketplace".
-4. The it loops through all the results and runs `fetchRepoExtensions()` or `getThemeRepos()`, which fetches a `manifest.json` file from the repo's root folder. If it finds one, we generate a card based on the info.
+4. The it loops through all the results and runs `getTaggedRepos()`, which fetches a `manifest.json` file from the repo's root folder. If it finds one, we generate a card based on the info.
```

Comment:
I'm pretty sure that this entire section needs updating since it's no longer as simple as it was before.

Reply:
I'm not completely sure but it seems like it still does the same thing in a nutshell.

Small change needed though
```suggestion
4. Then it loops through all the results and runs `getTaggedRepos()`, which fetches a `manifest.json` file from the repo's root folder. If it finds one, we generate a card based on the info.
```